src/Components/Movies.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+
+export class Movies extends Component {
+constructor() {
+    super();
+    this.state= {
+        movies: [],
+        selectedValue: '',
+        currmovie: {}
+    }
+}
+
+handleChange = async(e) => {
+    this.setState({
+        selectedValue: e.target.value
+    })
+    const { data } = await axios.get(`https://ghibliapi.herokuapp.com/films/${e.target.value}`);
+
+    this.setState({
+        currmovie: data
+    })
+}
+
+getMovies = async () => {
+    const { data } = await axios.get(`https://ghibliapi.herokuapp.com/films`)
+    this.setState({ movies: data })
+}
+
+componentDidMount() {
+    this.getMovies()
+}
+
+    render() {
+        const { movies, currmovie, selectedValue } = this.state
+        const options = movies.map(( movie, i ) => {
+            return <option
+            key={i}
+            value={movie.id}>
+                {movie.title}
+
+            </option>
+        })
+        return (
+            <div  style={{
+                backgroundColor: 'violet',
+                width: 'auto',
+                height: '500px'
+              }}
+              >
+                <h1>Select a Movie</h1>
+                <select
+                onChange={this.handleChange}
+                name=""
+                id=""
+                value={selectedValue}>
+                    <option></option>
+                    {options}
+                </select>
+                <h2>Title: {currmovie.title ? currmovie.title: null}</h2>
```

Comment:
You can also write this line:

currmovie.title ? currmovie.title: null;
as:
currmovie.title && currmovie.title; // This is called short circuiting